src/lib.rs

```diff
@@ -60,19 +33,6 @@ pub fn extract_pattern<'a>(toml_file: &'a Value, pattern: &str) -> Result<&'a Va
         })
 }
 
-pub fn load_toml_from_file(file_name: &str) -> Result<toml::Value> {
```

Comment:
not that I think anyone's really using this much, but this would constitute a breaking change that I'm not sure needs to happen. Maybe better to set this as deprecated and add a ticket to remove this in 0.2 release if anything

I'd also like to note that the new code is not really an accurate replacement for this since it gets rid of the file opening and toml parsing error messages

Reply:
I put it back in and marked it as deprecated.